ega-charts/localega/templates/lega-pvc.yaml

```diff
@@ -15,6 +15,8 @@ spec:
 {{ end }}
 ---
 {{ if and  .Values.persistence.enabled ( not .Values.postgres.persistence.existingClaim ) }}
+{{ if .Values.persistence.storageClass }}
+{{ if or ( eq "local-storage" .Values.persistence.storageClass ) }}
```

Comment:
Are these two conditionals necessary? If so, why? The second if statement is will always be go through once the first one has passed.

Reply:
Perhaps what you want is a `if and`?

Reply:
Nope, forgot to remove the `or`